lib/Alien/Base/ModuleBuild.pm

```diff
@@ -343,6 +347,10 @@ sub ACTION_alien_code {
     $self->config_data( system_provides => \%system_provides );
     return;
   }
+  
+  if ($ForceSystem) {
+    die "Requested $ENV{ALIEN_INSTALL_TYPE} install, but system package not detected."
```

Comment:
is it redundant to interpolate the ENV var here? Seems like system would be fine